gallery/gallery/lib/demos/reference/transformations_demo.dart

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show Vertices;
+import 'package:flutter/material.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+import 'transformations_demo_board.dart';
+import 'transformations_demo_edit_board_point.dart';
+import 'transformations_demo_gesture_transformable.dart';
+
+// BEGIN transformationsDemo#1
```

Comment:
Does the # 1 thing do something here?

Reply:
It allows concatenating code segments from different files, which I use here for 3/5 of them, I think